airflow-core/docs/authoring-and-scheduling/dynamic-task-mapping.rst

```diff
@@ -34,7 +34,7 @@ In its simplest form you can map over a list defined directly in your DAG file u
 
 If you want to see a simple usage of Dynamic Task Mapping, you can look below:
 
-.. exampleinclude:: /../../airflow/example_dags/example_dynamic_task_mapping.py
+.. exampleinclude:: /../../airflow-core/src/airflow/example_dags/example_dynamic_task_mapping.py
```

Comment:
Would all of these be better as 
```suggestion
.. exampleinclude:: ../../src/airflow/example_dags/example_dynamic_task_mapping.py
```
though maybe we want to avoid relative includes so it's easier when doc files move around, in which case

```suggestion
.. exampleinclude:: /../src/airflow/example_dags/example_dynamic_task_mapping.py
```

Either way -- going up two levels to go back in to `airflow-core/` seems redundant.

Reply:
I am afraid to touch it ... Every time I touch sphinx something unexpectedly blows up. 

But yeah let me try as separate small change for it and try it.

Reply:
Ah yeah. It blew-up.  The problem is that (for now until I fix doc generation in a follow up - some time in the coming weeks) - docs are still prepared in "docs" folder - they are just .gitignored and copied to the old place to make things work like they did before (I do the same with providers). Main thign is that currently sphinx generted all the docs for all doc packages  in the same tree and imports them from the same tree - so copying the rst files to the old place before building the docs seemed like the easiest way. I plan to fix it soon.

Reply:
Ah right, makes sense